docs/user_guides/Gufi For Coders/main.tex

```diff
@@ -0,0 +1,226 @@
+\documentclass{article}
+
+% Language setting
+% Replace `english' with e.g. `spanish' to change the document language
+\usepackage[english]{babel}
+
+% Set page size and margins
+% Replace `letterpaper' with`a4paper' for UK/EU standard size
+\usepackage[letterpaper,top=2cm,bottom=2cm,left=3cm,right=3cm,marginparwidth=1.75cm]{geometry}
+
+% Useful packages
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage[colorlinks=true, allcolors=blue]{hyperref}
+
+\title{GUFI: A Coder's Guide}
+\author{Braeden Slade}
+
+\begin{document}
+\maketitle
+
+\section{Introduction}
+Over the years, the amount of data we store and use has grown exponentially to the point that petabytes of storage is not uncommon. What used to be a simple task of accessing and sorting through information has been compounded into an arduous task with the size and scale of super-computing data centers. Being able to query data effectively, while also taking into account permissions becomes paramount into accomplishing daily tasks. This is what the Grand Unified File Index(GUFI) tool aims to accomplish. 
+
+\subsection{Background}
+This is accomplished by recreating the tree structure via indexing. Each directory contains an SQL database file that stores the metadata of the files as well as summary information for that directory and optionally summary information for the entire tree below that directory.
+
+\begin{figure} [h]
+\centering
+\includegraphics[width=0.5\textwidth]{gufi_structure.png}
+\caption{\label{fig:gufi\_structure}Layout and user interaction with GUFI}
+\end{figure}
+
+\clearpage
+
+\section{Understanding the data and how its stored}
+The data stored in these databases include filenames, access and creation dates, file attributes, and extended attributes. Depending on which table is viewed and what view is selected, there can be up to 59 attributes(columns) per entry(row)\\
```

Comment:
There **are** 59 attributes, some of which might be blank. More might be added in the future. Maybe change this to `there are currently 59 attributes`?

Reply:
Made the appropriate change